nrf_security/doc/backend_config.rst

```diff
@@ -7,27 +7,25 @@ Backend configurations and supported features
    :local:
    :depth: 2
 
-This section covers the configurations available when one or more nrf_security backends are enabled.
-This includes linking directly to the backend library or utilizing the Mbed TLS glue layer.
+This section covers the configurations available when either linking directly to a backend library or when using the Mbed TLS glue layer.
 
 .. _nrf_security_backend_config_multiple:
 
 Configuring multiple backends
 *****************************
 
-Different backends can support different cryptographic algorithms.
+Different backends support different cryptographic algorithms.
 The configuration options listed in subsequent sections are either *Glue*, *Shared*, or *Choice*.
 
 Glue
-   The configuration options will list `Glue` for the cryptographic algorithms that can be enabled in multiple backends at the same time.
+   Can be enabled in multiple backends simultaneously.
```

Comment:
This is a statement explaining the tables coming later. I don't want this part removed

Reply:
fixed. Algorithms is now stated.